documentation/src/main/asciidoc/topics/dependencies_maven/sb_bom.xml

```diff
@@ -0,0 +1,28 @@
+<properties>
+  <version.infinispan>{ispn_version}</version.infinispan>
+  <version.infinispan.starter>{sb_version}</version.infinispan.starter>
+</properties>
+
+<dependencyManagement>
+    <dependencies>
+       <dependency>
+           <groupId>org.infinispan</groupId>
+           <artifactId>infinispan-bom</artifactId>
+           <version>${version.infinispan}</version>
+           <type>pom</type>
+           <scope>import</scope>
+       </dependency>
+       <dependency>
+           <groupId>org.springframework.boot</groupId>
+           <artifactId>spring-boot-starter-parent</artifactId>
+           <version>${version.spring.boot}</version>
+           <type>pom</type>
+           <scope>import</scope>
+       </dependency>
+       <dependency>
+           <groupId>org.infinispan</groupId>
+           <artifactId>infinispan-spring-boot-starter</artifactId>
+           <version>${version.infinispan.starter}</version>
```

Comment:
if @karesti adds the starter to the infinipan-bom, you can remove the `<version>